tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -68,26 +73,77 @@ protected function setUp()
     {
         $this->handler = new AddPrefixCommandHandler();
         $this->io = new NormalizedLineEndingsIO('', self::$formatter);
+        $this->workingDirectory = getcwd();
         $this->tempDir = TestUtil::makeTempDir('php-scoper', __CLASS__);
 
         $filesystem = new Filesystem();
-        $filesystem->mirror(__DIR__.'/../Fixtures/original/dir', $this->tempDir);
+        $filesystem->mirror(__DIR__.'/../Fixtures/original', $this->tempDir);
     }
 
     protected function tearDown()
     {
+        chdir($this->workingDirectory);
         $filesystem = new Filesystem();
         $filesystem->remove($this->tempDir);
     }
 
     public function testAddPrefixToDirectory()
     {
-        chdir(dirname($this->tempDir));
+        chdir($this->tempDir);
+
+        $args = self::$command->parseArgs(
+            new ArgvArgs(
```

Comment:
Are you sure ? I'm getting a `Webmozart\Console\Api\Args\CannotParseArgsException: Not enough arguments.` when replacing the ArgvArgs with StringArgs